### DISCUSSION TITLE
[评论插件] 建议提供不同评论模块下已有评论数目预览

### DISCUSSION THREAD

**Discussion by SkysCrystal** [tangly1024/NotionNext#2004] [Ideas]
**为什么提出这个新的特性改动**
使用NotionNext搭建博客的用户大多开启了两种及以上的评论模块（我开了Cusdis和Utterance，准备尝试添加Waline）。在阅读完博客内容的时候，有时想看看评论区是否有相关内容讨论，就需要点击每个评论模块，等待加载，才能看到是否有留言。

**描述一下你推荐的解决方案**
是否可以提前在每个评论模块后，添加已有的评论数目预览，如：
![](https://raw.githubusercontent.com/SkysCrystal/Figurebed/master/img/202206161302772.png)
![](https://raw.githubusercontent.com/SkysCrystal/Figurebed/master/img/202206161302906.png)
可以提升评论阅读体验。

**描述一下你考虑过的其它替代解决方案**
不知道实现难度如何，暂无其他想法。

**补充说明**
暂无。

**Comment by tangly1024** [comment]
感谢提议，很好的意见👏，后续版本将会尝试添加此特性。

并非所有插件都支持获取评论数量、具体能实现的效果要依据实际开发情况而定。
